Formula/cloud-cli.rb

```diff
@@ -0,0 +1,15 @@
+class CloudCli < Formula
```

Comment:
Is the class name significant?

What is the install command that would be used?

Reply:
The install command will be `brew install cloud-cli`

Reply:
I would prefer `brew install snctl` since `cloud-cli` is just a repo name.